__tests__/app.test.js

```diff
@@ -31,4 +31,42 @@ describe("GET /api/categories", () => {
         });
       });
   });
+  test("status:404, responds with an error message when passed a url that does not exist", () => {
+    return request(app)
+      .get("/api/categoriees")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Route not found");
+      });
+  });
+});
+
+describe("GET /api/review/:reviewid", () => {
+  test("status:200, responds with a single review based on a given id", () => {
+    const review_id = 1;
+    return request(app)
+      .get(`/api/review/${review_id}`)
+      .expect(200)
+      .then((body) => {
+        expect(body.body).toEqual({
+          review_id: expect.any(Number),
+          title: expect.any(String),
+          category: expect.any(String),
+          designer: expect.any(String),
+          owner: expect.any(String),
+          review_body: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("status:404, responds with an error message when passed a url that does not exist", () => {
+    return request(app)
+      .get("/api/categoriees")
```

Comment:
this is not testing for an error to do with /api/reviews/review_id 
please alter this test :)